### PR TITLE
Adds missing headers for buckets and quantification methods

### LINKED ISSUE #7456: [Fill component] Add missing headers to go back from buckets and quantification methods
The buckets and quantification methods panes lack of a header to go back to the previous pane.

### PR DESCRIPTION
**Pull request by javierarce**
This PR adds missing headers to go back from the buckets and quantification panes of the Fill component:

screen shot 2016-05-23 at 12 37 29
screen shot 2016-05-23 at 12 37 23

Closes #7456

### CODE REVIEW

**Comment by javierarce** [comment]
CR: @xavijam 
CC: @piensaenpixel

File: lib/assets/locale/en.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
   "form-components": {
     "editors": {
       "fill": {
+        "quantification": 'Quantification',
```
Comment (xavijam) [inline lib/assets/locale/en.json]: double quotes, and below too :)

**Comment by xavijam** [comment]
Question, is it possible to hide quantification or bins in the component?

**Comment by javierarce** [comment]
@xavijam: it's not possible right now, but I can add that customization if you want.

**Comment by xavijam** [comment]
About my question: yep please :).
About the PR: 🚢 🇮🇹

**Comment by javierarce** [comment]
Great, I'll add a ticket for that! Thanks for the review :)

**Comment by piensaenpixel** [comment]
👍

**Pull request merged**